### PR TITLE
Adds missing descriptions again

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds some missing descriptions, where the item desc was null
## Why It's Good For The Game
Players finding an item that has no description is jarring.
## Changelog
:cl:
fix: added descriptions for holodeck floors and comp/mach frames
/:cl:

### CODE REVIEW

File: code/game/turfs/open/space/transit.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /turf/open/space/transit
 	name = "\proper hyperspace"
+	desc = "What is this, light-speed? We need to go to plad speed!"  // spaceballs was a great movie
```
Comment (OrionTheFox) [inline code/game/turfs/open/space/transit.dm:3]:
```suggestion
	desc = "What is this, light-speed? We need to go to plaid speed!"  // spaceballs was a great movie
```

File: code/modules/holodeck/turfs.dm [hunk 2]
```diff
@@ -154,12 +162,13 @@
...
 /turf/open/floor/holofloor/wood
 	icon_state = "wood"
+	desc = "Makes you feel at home."
 	tiled_dirt = FALSE
 
 /turf/open/floor/holofloor/snow
 	gender = PLURAL
 	name = "snow"
-	desc = "Looks cold."
+	desc = "The puffy snow clumps together to make a solid-looking floor, though still easily destroyed."
```
Comment (OrionTheFox) [inline code/modules/holodeck/turfs.dm:171]: considering it's a holo-floor, the description including mention of it being destructible feels wrong

Comment (etherware-novice) [reply code/modules/holodeck/turfs.dm:171]: fake destructible, like when u walk on snow irl and it kinda falls under ur foot

**Comment by ZephyrTFA** [comment]
Descriptions are not required for objects, and this isn't a 'fix'

**Comment by ZephyrTFA** [comment]
items we not supposed to have descriptions, they are optional

**Comment by JohnFulpWillard** [comment]
Descriptions aren't needed to be on everything, and personally I don't think messages like ``Hey look, it's the inside of a greytiders mind!`` is worth making the chatflood problem that much worse, can you explain your 'why this is good for the game' more?

**Comment by OrionTheFox** [comment]
For what its worth, I do think the console and machine frame descriptions are good - especially in that the machine one explains that it needs a machine board to be useful

**Comment by etherware-novice** [comment]
> Descriptions aren't needed to be on everything, and personally I don't think messages like ``Hey look, it's the inside of a greytiders mind!`` is worth making the chatflood problem that much worse, can you explain your 'why this is good for the game' more? 

it adds personality/informs, and players need to deliberately examine objects to see the description so i dont see where the chat flood problem comes from

File: code/game/machinery/constructable_frame.dm [hunk 3]
```diff
@@ -24,6 +25,7 @@
 
 /obj/structure/frame/machine
 	name = "machine frame"
+	desc = "The standard frame for most station appliances. Its appearance and function is controlled by the inserted board. "
```
Comment (JohnFulpWillard) [inline code/game/machinery/constructable_frame.dm:28]:
```suggestion
	desc = "The standard frame for most station appliances. Its appearance and function is controlled by the inserted board."
```

**Pull request merged**